src/main/resources/db/migration/V180__SubzonesAndMonitoringPlots.sql

```diff
@@ -0,0 +1,30 @@
+-- Rename the `tracking.plots` table to `tracking.planting_subzones`.
+ALTER TABLE tracking.plots RENAME TO planting_subzones;
+
+-- Rename `plot_id` reference from `tracking.plantings` to reference a `planting_subzone`.
+ALTER TABLE tracking.plantings RENAME plot_id TO planting_subzone_id;
+ALTER TABLE tracking.plantings
+    RENAME CONSTRAINT plantings_plot_id_fkey TO plantings_planting_subzone_id_fkey;
+
+-- Drop the `tracking.plot_populations` view. Recreate with name,
+-- `tracking.planting_subzone_populations`.
+DROP VIEW IF EXISTS tracking.plot_populations;
+CREATE VIEW tracking.planting_subzone_populations AS
+    SELECT planting_site_id, planting_subzone_id, species_id, SUM(num_plants) AS total_plants
+    FROM tracking.plantings
+    WHERE planting_subzone_id IS NOT NULL
+    GROUP BY planting_site_id, planting_subzone_id, species_id;
+
+-- Add a `tracking.monitoring_plots` table which references an id from `tracking.planting_subzones`.
+CREATE TABLE tracking.monitoring_plots (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    planting_subzone_id BIGINT NOT NULL REFERENCES tracking.planting_subzones ON DELETE CASCADE,
+    name TEXT NOT NULL,
+    full_name TEXT NOT NULL,
+    boundary GEOMETRY(MULTIPOLYGON),
```

Comment:
I'm firmly not sure yet. The initial shapefile we have uses (multi)polygons for monitoring plots. I _think_ it makes sense to store the monitoring plots as polygons instead of points, especially if we decide to convert away from UTM on storage, since the point-to-polygon conversion will be much easier in UTM. But I think that setting this as a polygon now doesn't presuppose or limit ourselves too much in the future.